### DISCUSSION TITLE
How to remove icons from node_modules ?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [rsuite/rsuite#2797] [Q&A]
Hi guys. I don't want to use rsuite icons, I have another option. Folder icons is 25MB on node_modules, is there a way to don't include them?

**Comment by SevenOutman** [comment]
`@rsuite/icons` is used by some rsuite components, e.g. Dropdown to display a caret, Table to display sorting directions. The icons package should be tree-shakable which means only the icons that's actually used will be included in your bundle, rather than the entire package. Furthermore, rsuite itself is tree-shakable too, that means even though Table component uses several icons, if you never use Table component, the Table component along with those icons won't be included in your bundle as well.
